ansible/templates/media.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Media component
+
+[Service]
```

Comment:
I removed the disabling of audio, since it doesn't hurt and makes Krockis more flexible.